po/pl.po

```diff
@@ -1953,20 +1951,18 @@ msgid "Internal parse error"
 msgstr "Wewnetrzny błąd parsera"
 
 #: rust/libnewsboat/src/fslock.rs:57
-#, fuzzy
 msgid "Failed to open lock file '%s': %s"
-msgstr "Nie udało się otworzyć pliku blokady: '%s' (%s)"
+msgstr "Nie udało się otworzyć pliku blokady: '%s': %s"
 
 #: rust/libnewsboat/src/fslock.rs:60 rust/libnewsboat/src/fslock.rs:83
 #: rust/libnewsboat/src/fslock.rs:115
 msgid "<filename containing invalid UTF-8 codepoint>"
 msgstr "<plik zawierający nieprawidłowy znacznik UTF-8>"
 
 #: rust/libnewsboat/src/fslock.rs:80
-#, fuzzy
 msgid "Failed to write PID to lock file '%s': %s"
 msgstr ""
-"Nie udało się zapisać indetyfikatora procesu do pliku blokady: '%s' (%s)"
+"Nie udało się zapisać identyfikatora procesu do pliku blokady: '%s': %s"
```

Comment:
Also looks like there are one too many colons. I now suspect this *is* intentional, but want to double-check with you :)

Reply:
Nah, it's a mistake, I somehow found a typo but not the additional `:` :D